test/e2e_node/system/types.go

```diff
@@ -164,3 +164,20 @@ var DefaultSysSpec = SysSpec{
 		},
 	},
 }
+
+var WindowsSysSpec = SysSpec{
```

Comment:
could we have a map or something that maps `GOOS` and `SysSpec` possibly?
That would make this more reusable I guess...